src/htmlrenderer.cpp

```diff
@@ -821,6 +904,23 @@ void HtmlRenderer::render(std::istream& input,
 	}
 }
 
+void HtmlRenderer::add_media_link(std::string& curline,
+	std::vector<LinkPair>& links, const std::string& url,
+	const std::string& media_url, unsigned int media_count, LinkType type)
+{
+	if (media_url.empty()) {
+		return;
+	}
+
+	std::string type_str = type2str(type);
+	int link_num = add_link(links,
```

Comment:
This should be an `unsigned int`, since that's what `add_link` returns.

Reply:
Oh, sorry, I didn't check the return type and copied the declaration in HtmlRenderer::render